src/main/java/com/activeviam/mac/cfg/impl/MacServerConfig.java

```diff
@@ -108,6 +115,7 @@
    */
   @Bean
   public Void startManager() {
+    DEBUG_PROPERTY = Boolean.parseBoolean(env.getProperty("server.debug"));
```

Comment:
I think it would be a good idea to make the property have `false` as a default value if it is absent from the config file